products/paas/shopware/fundamentals/secrets.md

```diff
@@ -12,20 +12,79 @@ Secrets stored in Vault are reusable, which means that you can reuse a secret va
 
 ## Creating a New Secret
 
-A secret is composed of a type, a key, and a value. The supported types are `env`, `buildenv`, and `ssh`. `env` is available at runtime in the application, `buildenv` is accessible during build processes, and `ssh` keys are for secure connections.
+A secret is composed of a type, a key, and a value. 
+Once created, it is assigned a unique `secret-id`, which is required for retrieving or deleting the secret.
+
+The supported types are `env`, `buildenv`, and `ssh`. `env` is available at runtime in the application, `buildenv` is accessible during build processes, and `ssh` keys are for secure connections.
 
 To create a secret:
 
 ```sh
 sw-paas vault create
 ```
 
-### Deleting a Secret
+---
+
+## Retrieving a Secret
+
+To retrieve an existing secret from the Vault, you **must specify the secret ID** using the `--secret-id` flag:
+
+```sh
+sw-paas vault get --secret-id SECRET-ID
+```
+
+> ℹ️ You can obtain the `SECRET-ID` by running:
+>
+> ```sh
+> sw-paas vault list
+> ```
+
+---
 
-To delete an existing secret:
+## Deleting a Secret
 
-**Usage:**
+To delete a secret from the Vault, also use the `--secret-id` flag:
 
 ```sh
-sw-paas vault delete
+sw-paas vault delete --secret-id SECRET-ID
 ```
+
+> ⚠️ **Warning:** Deleting a secret is permanent. Ensure the secret is no longer in use before removing it.
```

Comment:
Please use the docs specific markdown for info, warnings etc.

See here: https://github.com/shopware/docs/blob/f42e8a87635505da427ae59ad5dcb1ecf4751b37/products/pwa.md?plain=1#L14